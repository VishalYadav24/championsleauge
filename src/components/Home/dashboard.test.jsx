import { Dashboard } from '@mui/icons-material';
import renderer from 'react-test-renderer';

it('renders correctly when there are no items', () => {
    const tree = renderer.create(<Dashboard />).toJSON();
    expect(tree).toMatchSnapshot();
  });