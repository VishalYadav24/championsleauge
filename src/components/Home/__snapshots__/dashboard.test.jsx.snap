// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<svg
  aria-hidden={true}
  className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
  data-testid="DashboardIcon"
  focusable="false"
  viewBox="0 0 24 24"
>
  <path
    d="M3 13h8V3H3v10zm0 8h8v-6H3v6zm10 0h8V11h-8v10zm0-18v6h8V3h-8z"
  />
</svg>
`;
